src/OmniSharp.Host/Services/AssemblyLoader.cs

```diff
@@ -20,18 +20,17 @@ public AssemblyLoader(ILoggerFactory loggerFactory)
 
         public Assembly Load(AssemblyName name)
         {
-            Assembly result;
+            Assembly result = null;
             try
             {
                 result = Assembly.Load(name);
+                _logger.LogTrace($"Assembly loaded: {name}");
```

Comment:
Looks like the only consumer of this method already checks for null; sounds good.